src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1186,9 +1196,27 @@ int wm_vuldet_send_cve_report(vu_report *report) {
         cJSON *j_package = NULL;
         cJSON *j_cvss = NULL;
 
+        cJSON_AddItemToObject(alert, "vulnerability", alert_cve);
```

Comment:
Although this item was being added some lines below, the `alert_cve` and any other JSON object should be added immediately after being created. In this way, we avoid any kind of memory leak if the code goes to `end`, because the object will be within the `alert` JSON object, and will be recursively destroyed at the end of the function.